src/css/base.scss

```diff
@@ -61,3 +60,13 @@ display: none;
   width: 30%;
```

Comment:
if you haven't already, I suggest using `object-fit: cover` on your card images if you want them to retain their original ratio, but have different dimensions than the original file. The images are pretty much all different aspect ratios.